modules/lastseen.cpp

```diff
@@ -128,6 +130,7 @@ class CLastSeenMod : public CModule {
         return false;
     }
 
+    // Provides GUI to configure this module by adding a widget to user page in webadmin.
```

Comment:
Should I fix the commit messages and create another pull request/close this one?